sql-gremlin/src/main/resources/error-messages_en.properties

```
@@ -79,3 +79,4 @@ UNSUPPORTED_LITERAL_EXPRESSION="Unsupported: Raw literal expressions without any
 CANNOT_JOIN_DIFFERENT_EDGES="Error: Cannot perform JOIN on two edges that are not the same (%s != %s)."
 UNSUPPORTED_OPERAND_TYPE="Unsupported operand type %s, cannot rename column."
 UNRECOGNIZED_TYPE="Error, unrecognized type: '%s'."
+UNSUPPORTED_BASIC_LITERALS="Unsupported: Unsupported clause - Only basic literal comparisons are supported inside WHERE and HAVING at this time."
```

Add missing new line
```suggestion
UNSUPPORTED_BASIC_LITERALS="Unsupported: Unsupported clause - Only basic literal comparisons are supported inside WHERE and HAVING at this time."

```